cmd/containerd/main_windows.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google.golang.org/grpc"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/containerd/execution"
+	"github.com/urfave/cli"
+)
+
+func appendPlatformFlags(flags []cli.Flag) []cli.Flag {
+	return flags
+}
+
+func processRuntime(ctx context.Context, runtime string, root string) (execution.Executor, error) {
+	// TODO
+	return nil, nil
```

Comment:
You're missing my point - let's do this the same way as the docker port to Windows in incremental baby steps. Having this return nothing rather than panic lets me get to the next big chunk of code and at least have it running but not really doing anything. That is by FAR the easiest way to address the port. Otherwise, I will just have to keep removing the panic.